crates/store/re_log_encoding/src/rrd/footer/instances.rs

```diff
@@ -0,0 +1,16 @@
+/// This is the message type that is passed in the footer of RRD streams.
```

Comment:
That's inaccurate. The ::End message carries the payload, while the stream footer references the ::End message. Is there any place where we already have some kind of holistic doc for this?